### ISSUE TITLE
"Facebook is not available on this browser"

### ISSUE THREAD

**Issue by gab12** [gsurrel/FacebookNotifications#42]
when i open the app the webview shows the message "Facebook is not available on this browser".
So it is impossible to log in.

**Comment by gsurrel** [comment]
Indeed, I can reproduce. I've the impression it *could* be fixed using this user-agent:

    Mozilla/5.0 (Linux; Android 7.0; Pixel C Build/NRD91D; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/53.0.2785.124 Safari/537.36 [FB_IAB/FB4A;FBAV/98.0.0.18.70;]

**Comment by gab12** [comment]
Do you think a new release with the fix is possible? I use your app a lot!

**Comment by gsurrel** [comment]
Hi! Sorry about that delay, I've totally forgotten about that problem.

I've reinstalled all the Android development tools, spent a few hours upgrading everything so it build the app, and tried that other user-agent mentioned, but unfortunately, Facebook made enough changes everywhere that it doesn't work anymore.

Maybe I can give you a build where you can set the user-agent you want if you want to try out some things?

**Comment by gsurrel** [comment]
I created a new build here: 
[notification-for-facebook-useragent.zip](https://github.com/user-attachments/files/17535762/notification-for-facebook-useragent.zip)

I've set the user-agent that I've used in the first message. It works for the login, but it's stuck afterwards. I have the impression that Facebook has simply removed all the "light" version of the website I was using, breaking everything I relied on.

Maybe it's the end of this project.
